src/codeu/chat/server/fileWriter.java

```diff
@@ -0,0 +1,98 @@
+package codeu.chat.server;
+
+import java.util.concurrent.BlockingQueue;
+import java.lang.Runnable;
+
+import codeu.chat.common.Writeable;
+import codeu.chat.util.Serializers;
+
+import java.io.FileOutputStream;
+import java.io.FileInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import codeu.chat.common.User;
+
+import codeu.chat.util.Logger;
+
+public class fileWriter implements Runnable {
```

Comment:
For consistency, this should be called 'FileWriter'

Only instance variables and methods begin with lower case letters.